src/main/java/com/commercetools/pspadapter/notification/processor/impl/NotificationProcessorContainerImpl.java

```diff
@@ -19,8 +19,14 @@
 
     @Autowired
     public NotificationProcessorContainerImpl(@Nonnull PaymentSaleCompletedProcessor paymentSaleCompletedProcessor,
+                                              @Nonnull PaymentSaleDeniedProcessor paymentSaleDeniedProcessor,
+                                              @Nonnull PaymentSaleRefundedProcessor paymentSaleRefundedProcessor,
+                                              @Nonnull PaymentSaleReversedProcessor paymentSaleReversedProcessor,
                                               @Nonnull DefaultNotificationProcessor defaultNotificationProcessor) {
         this.notificationProcessorsMap.put(NotificationEventType.PAYMENT_SALE_COMPLETED.getPaypalEventTypeName(), paymentSaleCompletedProcessor);
+        this.notificationProcessorsMap.put(NotificationEventType.PAYMENT_SALE_DENIED.getPaypalEventTypeName(), paymentSaleDeniedProcessor);
```

Comment:
do you think we could try to use _List Of Beans_ Spring feature to avoid manual adding?

Pls, try to follow [this guide from Jürgen Höller](https://www.youtube.com/watch?v=alYxVgzfkW8&feature=youtu.be&t=2070). Ask me if you have any questions.

Reply:
https://github.com/commercetools/commercetools-paypal-plus-integration/pull/64/commits/055aab789aa34a29a7a62860a84c3f012ca7eb84